docs/gettingstarted/k8s-microk8s.md

```diff
@@ -1,42 +1,34 @@
 
-# Install MicroK8s
-The SC4S deployment model with Microk8s uses specific features of this distribution of k8s. 
-While this may be reproducible with other distributions such an undertaking requires more advanced
-awareness and responsibility for the administrator.
+# Install and configure SC4S with Kubernetes
+Splunk provides an implementation for SC4S deployment with MicroK8s using a single-server MicroK8s as the deployment model. Clustering has some tradeoffs and should be only considered on a deployment-specific basis.
 
-* (metalLB) ensure source IP is preserved
-* Bring any operating system (window/centos/rhel/ubuntu/debian)
+You can independently replicate the model deployment on different distributions of Kubernetes. To do this you must have Administrator privleges, and the process will require advanced understanding and responsibility on your part.
```

Comment:
I think the original piece mean that the administrator is expected to have advanced awareness and responsibility, not that Administrator privileges are required